### DISCUSSION TITLE
problem with both the CLI and WebUI

### DISCUSSION THREAD

**Discussion by Mou122** [BrokenSource/DepthFlow#40] [General]
Thank you for your hard work!
I am experiencing a problem with both the CLI and WebUI. In the CLI, it disconnects when I type the prompt input -i. The same issue occurs in the WebUI. Could you please assist me with this?
![Capture](https://github.com/user-attachments/assets/f85fded4-c68d-4d1c-b0e5-d2899950c986)

**Comment by ashenvi-123** [comment]
Same issue here. Please help!

**Reply by Tremeschin** [reply]
The release v0.7 changed a lot of things since the original question was asked (the version you're likely running)

(sorry for long-ish answer, a couple of ideas and insights)

One dude appeared on Discord today with similar issues on Windows, it seems to me that the code was segfaulting when reading the images out of OpenGL. I only have limited hardware of different platforms to test it, but the first thing I'd advise trying is to [ensure the right gpu is being used](https://brokensrc.dev/about/faq/#depthflow-shaderflow) (at least for nvidia hybrid systems)

This will be an option on a future release to disable it, but I'm enabling [turbopipe](https://github.com/BrokenSource/TurboPipe) by default when rendering the videos, the mileage with it was great and only a few users seem to have issues with it but I didn't understand the pattern yet.

It can be disabled by either rendering with the CLI with `depthflow input (...) main (...) --noturbo` or shift+right clicking an empty space in windows explorer, "open in powershell", then run `$env:NOTURBO=1` and then running `.\depthflow-cpu-windows-amd64-latest.exe gradio`

But all I said doesn't matter if the console shows any other errors (which if it just closes suddenly, running with the last instructions should hold it open so you can see any logs); in which case, could you screenshot the terminal if there's more information on what went wrong?
